lake/pools/config.go

```diff
@@ -27,6 +28,9 @@ func NewConfig(name string, sortKey order.SortKey, thresh int64, seekStride int)
 	if seekStride == 0 {
 		seekStride = data.DefaultSeekStride
 	}
+	if sortKey.IsNil() {
+		sortKey = order.NewSortKey(order.Desc, field.DottedList("ts"))
+	}
```

Comment:
Nit: Move this ahead of `if thresh == 0 {...}` so these remain in parameter order.